projects/schematics/src/add-pwa/index_spec.ts

```diff
@@ -64,6 +64,64 @@ describe('Spartacus Schematics: add-pwa', () => {
       .toPromise();
   });
 
+  describe('should add PWA properly', () => {
+    it('should not touch app.module.ts', async () => {
+      const tree = await schematicRunner
+        .runSchematicAsync(
+          'ng-add',
+          {
+            ...defaultOptions,
+            name: 'schematics-test',
+            pwa: true,
+          },
+          appTree
+        )
+        .toPromise();
+
+      const appModule = tree.readContent(
+        '/projects/schematics-test/src/app/app.module.ts'
+      );
+      expect(appModule).toMatchSnapshot();
+
+      const packageJson = tree.readContent('/package.json');
+      expect(packageJson).toMatchSnapshot();
+    });
+
+    it('should modify package.json', async () => {
```

Comment:
`query:` is it not already checked in the previous test?

Reply:
Good catch, thanks!
It was a leftover, I removed it now.